pkg/gofr/datasource/README.md

```diff
@@ -69,21 +69,19 @@ Therefore, GoFr utilizes a pluggable approach for new datasources by separating
 
 ## Supported Datasources
 
-| Datasource      | Health-Check | Logs | Metrics | Traces | As Driver |
-|-----------------|--------------|------|---------|--------|-----------|
-| MySQL           | ✅            | ✅    | ✅       | ✅      |           |
-| Redis           | ✅            | ✅    | ✅       | ✅      |           |
-| PostgreSQL      | ✅            | ✅    | ✅       | ✅      |           |
-| MongoDB         | ✅            | ✅    | ✅       | ✅      | ✅         |
-| SQLite          | ✅            | ✅    | ✅       | ✅      |           |
-| BadgerDB        | ✅            | ✅    | ✅       | ✅      | ✅         |
-| Cassandra       | ✅            | ✅    | ✅       | ✅      | ✅         |
-| ClickHouse      |              | ✅    | ✅       | ✅      | ✅         |
-| FTP             |              | ✅    |         |        | ✅         |
-| SFTP            |              | ✅    |         |        | ✅         |
-| Solr            |              | ✅    | ✅       | ✅      | ✅         |
-| Dgraph          | ✅            | ✅    | ✅       | ✅      |           |
-| Azure Event Hub |              | ✅    | ✅       |        | ✅         |
-| OpenTSDB        | ✅            | ✅    |         | ✅      | ✅         |
-
-
+| Datasource     | Health-Check | Logs | Metrics | Traces | As Driver |
+|----------------|:------------:|:----:|:-------:|:------:|:---------:|
+| MySQL          | ✅         | ✅    | ✅    | ✅      |           |
+| REDIS          | ✅         | ✅    | ✅    | ✅      |           |
+| PostgreSQL     | ✅         | ✅    | ✅    | ✅      |           |
+| MongoDB        | ✅         | ✅    | ✅    |  ✅      | ✅         |
+| SQLite         | ✅         | ✅    | ✅    | ✅      |           |
+| BadgerDB       | ✅         | ✅    | ✅    |    ✅    | ✅         |
+| Cassandra      | ✅         | ✅    | ✅    |  ✅      | ✅         |
+| Clickhouse     |            | ✅    | ✅    |   ✅     | ✅         |
+| FTP            |            | ✅    |       |         | ✅         |
+| SFTP           |            | ✅    |       |          | ✅         |
+| Solr           |            | ✅    | ✅    |   ✅     |   ✅     |
+| DGraph         | ✅         | ✅    | ✅    |   ✅     |         |
+| Azure Eventhub |            | ✅    | ✅    |          |✅     |
```

Comment:
Why rolling back my changes?

c3ab23c816028840b5112b7e946b7970bfa83fbd

I detected at least `Event Hub`, `Redis`

It looks like a conflict badly resolved

Reply:
@danysz Please revert these changes.